codebuild/test-aws-java-sdk-v2.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+#this build spec assumes the ubuntu aws/codebuild/java:openjdk-8 image
+phases:
+  install:
+    runtime-versions:
+      java: "corretto8"
+    commands:
+      - add-apt-repository ppa:openjdk-r/ppa
+      - add-apt-repository ppa:ubuntu-toolchain-r/test
+      - apt-get update -y
+      - apt-get install gcc-7 cmake ninja-build -y
+      - git submodule update --init --recursive
+      - git clone https://github.com/aws/aws-sdk-java-v2.git
+      # change the version from SDK to local SNAPSHOT version
+      - sed -i 's/<awscrt.version>.*<\/awscrt.version>/<awscrt.version>1.0.0-SNAPSHOT<\/awscrt.version>/g' pom.xml
```

Comment:
shouldn't this be ~pom.xml~ -> `aws-sdk-java-v2/pom.xml`?
aren't we still in the `aws-crt-java/` folder?

Reply:
oooops